test/cli_ps_test.go

```diff
@@ -71,6 +72,139 @@ func (suite *PouchPsSuite) TestPsWorks(c *check.C) {
 
 }
 
+// TestPsFilterInvalid tests "pouch ps -f" invalid
+func (suite *PouchPsSuite) TestPsFilterInvalid(c *check.C) {
+	result := command.PouchRun("ps", "-f", "foo")
```

Comment:
Next time, we can use the table-driven test to repeat the same thing.

Reply:
^-^